app_grim_reefer/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#-------------------------------------------------------------------------------
+# Backplane Power Module Application
```

Comment:
```suggestion
# GRIM Reefer Application
```

Reply:
fixed in [314d58d](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/314d58d64e4811e7a7bd31817aa276b685ae3c5b)